database/seeds/PostsTableSeeder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PostsTableSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        \App\Post::firstOrCreate(['slug' => 'post1'], [
+            'author_id' => 1,
+            'title'     => 'Title1',
+            'content'   => 'Content1',
+            'excerpt'   => 'Excerpt1',
+            'status_id' => 2
```

Comment:
Use `PostStatus` constants here. It should be clear what each status mean. That's the main idea of using constants.

Reply:
Fixed.